README.md

```diff
@@ -45,7 +45,9 @@ import './jestGlobalMocks'; // browser mocks globally available for every test
   "transform": {
     "^.+\\.(ts|js|html)$": "<rootDir>/node_modules/jest-preset-angular/preprocessor.js"
   },
-  "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|js)$",
+  "testMatch": [
+    "(/__tests__/.*|\\.(test|spec))\\.(ts|js)$"
```

Comment:
This is wrong. Here's the default one (where I just found a bug XD):
```
[
  '**/__tests__/**/*.js?(x)', 
  '**/?(*.)(spec|test).js?(x)' 
]
```
and we rather want to change it to:
```
[
  '**/__tests__/**/*.+(ts|js)?(x)', 
  '**/?(*.)+(spec|test).+(ts|js)?(x)' 
]
```

Reply:
oh those 2 are default from Jest 👍